build.sh

```diff
@@ -24,17 +24,21 @@ if [ "$EVENT_TYPE" = "cron" ]  || [ "$DEV" = "true" ]; then
     DJ_MM3_REF=$(python get_latest_ref.py mailman/django-mailman3)
     MM3_HK_REF=$(python get_latest_ref.py mailman/mailman-hyperkitty)
 
-    # Build the mailman-core image.
-    $DOCKER build -f core/Dockerfile.dev \
+	for value in amd64 arm32v7 arm64v8
```

Comment:
```suggestion
	for arch in amd64 arm32v7 arm64v8
```
Can you also rename the variables used below to `$arch` instead and other loops?